### PR TITLE
Vaishnavi_Assignment_1_Done

### PR DESCRIPTION
**Pull request by VaishnaviVJun20**
image

### CODE REVIEW

File: src/vaishnavi_SeleniumBasics/VerifyFirstWebPage.java [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+		else
+			System.out.println("You are on a page of title: " + driver.getTitle());
+
+		// Filling all the details of form
+		driver.findElement(By.id("first name")).sendKeys("Vaishnavi");
+		driver.findElement(By.id("last name")).sendKeys("Vaidya");
+		driver.findElement(By.id("E-mail")).sendKeys("Vaishnavivaidya06@gmail.com");
+		driver.findElement(By.id("Company Name")).sendKeys("Atos-Syntel");
+
+		WebElement maleRadioButton = driver.findElement(By.id("male"));
```
Comment (KrishnaKTechnocredits) [inline src/vaishnavi_SeleniumBasics/VerifyFirstWebPage.java:29]: You missed this validation,  for Gender radio button if ‘Female’ radio button is by default selected then select ‘Male’   radio button and vice versa.

File: src/vaishnavi_SeleniumBasics/VerifyFirstWebPage.java [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+		maleRadioButton.click();
+
+		WebElement experience = driver.findElement(By.id("1-3"));
```
Comment (KrishnaKTechnocredits) [inline src/vaishnavi_SeleniumBasics/VerifyFirstWebPage.java:32]: You should only store any value in webElement when you want to perform more than 1 action. 
But here you are only performing one action which click() so you should write like this:
 driver.findElement(By.id("1-3")).click();

File: src/vaishnavi_SeleniumBasics/VerifyFirstWebPage.java [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+		WebElement dreamCompany = driver.findElement(By.id("google1"));
+		dreamCompany.click();
+		WebElement termsAndCondition = driver.findElement(By.id("check"));
+		termsAndCondition.click();
+
+		// Reseting the values of this form
+		driver.findElement(By.id("reset12")).click();
+
+		// Checking if all the radio buttons and check boxes reset is successfully done
+		if (maleRadioButton.isSelected() == false)
```
Comment (KrishnaKTechnocredits) [inline src/vaishnavi_SeleniumBasics/VerifyFirstWebPage.java:46]: Verify Reset works properly on your text field, radio button and checkbox.

here you missed validation for text field.

File: src/vaishnavi_SeleniumBasics/VerifyFirstWebPage.java (hunk 1 above)
Comment (VaishnaviVJun20) [reply src/vaishnavi_SeleniumBasics/VerifyFirstWebPage.java]: -- Added suggested condition to check default selected button.

File: src/vaishnavi_SeleniumBasics/VerifyFirstWebPage.java (hunk 2 above)
Comment (VaishnaviVJun20) [reply src/vaishnavi_SeleniumBasics/VerifyFirstWebPage.java:50]: -- Actually I am using this to verify the reset condition later as well , Please let me know if still I should remove this.

File: src/vaishnavi_SeleniumBasics/VerifyFirstWebPage.java (hunk 3 above)
Comment (VaishnaviVJun20) [reply src/vaishnavi_SeleniumBasics/VerifyFirstWebPage.java]: -- Added one condition for text field as well

Comment (VaishnaviVJun20) [reply src/vaishnavi_SeleniumBasics/VerifyFirstWebPage.java]: -- Thank you for all the suggestions. :)

File: src/vaishnavi_SeleniumBasics/VerifyFirstWebPage.java (hunk 2 above)
Comment (KrishnaKTechnocredits) [reply src/vaishnavi_SeleniumBasics/VerifyFirstWebPage.java:50]: No dear.

**Pull request merged**